other/quiz/quiz_questions.js

```diff
@@ -197,7 +197,11 @@ const pokemonType = () => new Promise(resolve => {
   (async () => {
     const pokemon = getRandomPokemon();
     const types = pokemon.type.map(t => PokemonType[t]);
-    const answer = new RegExp(`^\\W*(${types.join('\\W*')}|${types.reverse().join('\\W*')})\\b`, 'i');
+    let memeAnswer = null;
```

Comment:
I think setting this to a blank string resolves it though?
```suggestion
    let memeAnswer = '';
```